### ISSUE TITLE
Keyboard navigation in events list in Live Data views not working

### ISSUE THREAD

**Issue by lnlp** [TheThingsNetwork/lorawan-stack#3719]
#### Summary
Keyboard navigation does not work for navigating the events list in Live Data views.
It is not possible to scroll through the list messages with common navigations keys like cursor keys, (ctrl-)home, (ctrl-)end, PgUp and PgDn keys.
One can only navigate with mouse actions. There is no keyboard support.

#### Steps to Reproduce
In the Console open a Live Data window for gateway, application or device.
Wait until there is a long list of messages that don't fit on screen (e.g. a list three times the height of the window).
Activate the message list by clicking on it. Now try to navigate / scroll through the messages with the regular navigation keys. It just does not work. 

#### What do you see now?
See Steps to reproduce.

#### What do you want to see instead?
Be able to navigate, scoll, page, jump to first, jump to last events with navigation keys as is common in most UI's.

#### Environment
[The Things Network V3 Console](https://eu1.cloud.thethings.network/console/).

#### How do you propose to implement this?
Add keyboard navigation support.

**Comment by htdvisser** [comment]
I don't think this is a bug, just missing functionality. @kschiffer please add bug label if you think otherwise.

**Comment by kschiffer** [comment]
One could say that this is an accessibility bug, but I'd also rather file this as feature request. @asmulko can you take a look at this?

**Comment by kschiffer** [comment]
We will be working on focusing and cycling through individual event rows with the keyboard, as well as toggling their details using space/enter. Also PageUp and PageDown should jump to the first/last event.

Would that satisfy your needs @lnlp ?

**Comment by lnlp** [comment]
> Would that satisfy your needs @lnlp ?

Yes I think so.

**Comment by kschiffer** [comment]
Note for ourselves: this should also include the possibility to close the event details using the escape key #3717
